site/docs/v1/tech/oauth/endpoints.adoc

```diff
@@ -786,6 +786,15 @@ The refresh token as described by https://tools.ietf.org/html/rfc6749#section-1.
 This field will be returned in the response when `offline_access` scope was requested unless refresh tokens have been disabled for this application.
 See `oauthConfiguration.generateRefreshTokens` in the link:/docs/v1/tech/apis/applications[Application API] or the `Generate refresh tokens` toggle in the FusionAuth UI when editing an application.
 
+[field]#refresh_token_id# [type]#[UUID]#::
+The unique ID of the refresh token. This ID can be used to retrieve or revoke a specific refresh token when using the link:/docs/v1/tech/apis/jwt[Refresh Token API].
++
+This field will be returned in the response when `offline_access` scope was requested unless refresh tokens have been disabled for this application.
+See `oauthConfiguration.generateRefreshTokens` in the link:/docs/v1/tech/apis/applications[Application API] or the `Generate refresh tokens` toggle in the FusionAuth UI when editing an application.
+
+[field]#scope# [type]#[String]#::
+The space-separated list of scopes that were granted to this access token. This will only be present for grant types of `password` and `authorization_code`.
```

Comment:
```suggestion
The space-separated list of scopes allowed by the user during the grant which generated this access token. This will only be present for Authorization Code and Password grants.
```

I think that is a bit clearer.